src/DropdownButton/DropdownButton.tsx

```diff
@@ -23,7 +23,7 @@ export interface Props {
   className?: string;
   disabled?: boolean;
   href?: string;
-  label: string;
+  label: React.ReactNode;
```

Comment:
sanity check: this will still accept a string, correct?

Reply:
It still accepts strings, yep!